src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2857,27 +2854,40 @@ namespace wallet_rpc
   };
 
   LOKI_RPC_DOC_INTROSPECT
-  // Buy a Loki Name Service mapping. Specifying `owner` is optional and defaults to the purchasing wallet if empty or not specified. The `owner` is a ED25519 public key, derived from the wallets spend key.
-  struct COMMAND_RPC_BUY_LNS_MAPPING
+  struct COMMAND_RPC_LNS_BUY_MAPPING
   {
+    static constexpr const char *description =
+R"(Buy a Loki Name System mapping. Loki Name System allows multiple owners that are authorised to update the underlying mapping. By default if no owner is specified, the purchasing wallet's public spend key is set as the owner.
+  - For Session, the recommended owner is the ed25519 public key of the user's Session ID set to ed_owner
+
+In future, support for mappings on Blockchain wallets and Lokinet addresses will be available. Tentatively,
+  - for Wallets, the recommended owner is the monero ed25519 public key of the user's wallet spend key set to loki_owner
+  - for Lokinet, the recommended default owner is the ed25519 public key of the user's Lokinet key set to ed_owner
```

Comment:
I don't think the "for Lokinet" part here really makes sense.  Lokinet doesn't really expose your keys for something like this; it makes more sense to keep it as the wallet keys here, as well: so Session recommends the session public key, and everything else recommends the wallet public key.